public/static/docs/command-reference/update.md

```diff
@@ -27,13 +27,13 @@ Note that import stages are considered always locked, meaning that if you run
 `dvc repro`, they won't be updated. `dvc update` is the only command that can
 update them.
 
-Another detail to note is that when the `--rev` (revision) option of
-`dvc import` has been used to create an import stage, DVC is not aware of what
-kind of
-[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References) this
-is, for example a branch or a tag. For typically static references (e.g. tags),
-or for SHA commits, `dvc update` will not have any effect on the import. Refer
-to the
+Another detail to note is that when the `--rev`
+([revision](https://git-scm.com/docs/revisions)) option of `dvc import` has been
+used to create an import stage, DVC is not aware of what kind of
+[Git reference](https://git-scm.com/book/en/v2/Git-Internals-Git-References) has
```

Comment:
all paragraph is quite complicated, would update it the way we discuss in the regular update import improvement

Reply:
OK, got it. It's a tricky one though, how about
```markdown
`dvc update` will not have an effect on import stages created with `dvc import`
if option `--rev` is used with a specific commit (`rev` does not change). (In
these cases, you may
[re-import](/doc/command-reference/import#example-fixed-revisions-re-importing)
a different commit instead.)
```
? (Updated to this for now in 57ff7362.)